src/sagas/duplicatesSaga.js

```diff
@@ -0,0 +1,51 @@
+import {
+    fetchDuplicatesError,
+    fetchDuplicatesSuccess,
+    ignoreDuplicatesError,
+    ignoreDuplicatesSuccess,
+    mergeDuplicatesError,
+    mergeDuplicatesSuccess
+} from "../actions/duplicatesEvents";
```

Comment:
import * as ... it helps to see where something comes from do it for all imports